cbv3_django_prototype/cbv3_django_prototype/resources/views.py

```diff
@@ -1,3 +1,14 @@
-from django.shortcuts import render
+from .models import Resource
+from django.shortcuts import render, get_object_or_404
+from django.contrib.postgres.fields import JSONField
+from .serializers import ResourceSerializer
+from rest_framework import filters, viewsets
+from rest_framework.response import Response
+
+
+class ResourceListView(viewsets.ModelViewSet):
+    queryset = Resource.objects.all()
+    serializer_class = ResourceSerializer
+    filter_backends = [filters.SearchFilter]
+    search_fields = ['id', 'tags']
```

Comment:
We probably want to also search by `title` and `description`.

Reply:
Agreed.  Will add on my next revision.

Reply:
search_fields = ['id', 'media_type', 'title', 'description', 'tags']